src/evaluators/scatter/PHAL_ScatterSideEqnResidual_Def.hpp

```diff
@@ -113,29 +113,84 @@ postRegistrationSetup(typename Traits::SetupData d,
   d.fill_field_dependencies(this->dependentFields(),this->evaluatedFields());
 }
 
+template<typename EvalT, typename Traits>
```

Comment:
It's probably safer to error out if the mesh is not extruded.

Reply:
True.